scripts/rename-galois-to-katana.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+MODE=${1:-}
+
+if [[ ${MODE} == "contents" ]]; then
+  while read -d '' filename; do
+    # Otherwise perl -i will clobber symlinks
+    file="$(readlink -e -- "${filename}")"
+
+    # C++ namespaces
+
+    # (?<!...) negative lookback
+    perl -p -i \
+      -e 's/galois::/katana::/g;' \
+      -e 's/namespace\s+galois/namespace katana/g;' \
+      \
+      -e 's/katana::runtime/katana/g;' \
+      -e 's/typename\s+runtime:://g;' \
+      -e 's/(?<![\w:])runtime:://g;' \
+      -e 's/namespace\s+runtime.*{.*//g;' \
+      -e 's/}.*namespace runtime.*//g;' \
+      \
+      -e 's/katana::worklists/katana/g;' \
+      -e 's/typename\s+worklists:://g;' \
+      -e 's/(?<![\w:])worklists:://g;' \
+      -e 's/namespace\s+worklists.*{.*//g;' \
+      -e 's/}.*namespace worklists.*//g;' \
+      \
+      -e 's/katana::graphs/katana/g;' \
+      -e 's/typename\s+graphs:://g;' \
+      -e 's/(?<![\w:])graphs:://g;' \
+      -e 's/namespace\s+graphs.*{.*//g;' \
+      -e 's/}.*namespace graphs.*//g;' \
```

Comment:
Same comment as before. Why not do the same for `substrate`?

Reply:
flattened substrate too